azure-container-apps/ReadMe.md

```diff
@@ -1,337 +1,349 @@
-This deployment example describes how to deploy 1Password SCIM Bridge as an app on Azure's [Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service.
+_Learn how to deploy 1Password SCIM Bridge on the [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service._
 
-The deployment consists of two [containers](https://learn.microsoft.com/en-us/azure/container-apps/containers), the SCIM bridge container and a Redis container and an [ingress](https://learn.microsoft.com/en-us/azure/container-apps/ingress-overview) for the SCIM bridge container.
+This deployment consists of two [containers](https://learn.microsoft.com/en-us/azure/container-apps/containers): One for the SCIM bridge and another for Redis. There's also an [ingress](https://learn.microsoft.com/en-us/azure/container-apps/ingress-overview) for the SCIM bridge container. There are a few benefits to deploying 1Password SCIM Bridge on Azure Container Apps:
 
-## In this folder
+- For standard deployments, the service will host your SCIM bridge for ~$16 USD/month (as of January 2024). Container Apps pricing is variable based on activity, and you can learn more on [Microsoft's pricing page](https://azure.microsoft.com/en-us/pricing/details/container-apps/).
+- You don't need to manage a DNS record. Azure Container Apps automatically provides a unique one for your SCIM bridge domain.
+- Azure Container Apps automatically handles TLS certificate management on your behalf.
+- The SCIM bridge can be deployed directly to Azure from the Portal or via the command line in your local terminal. There is no requirement to clone this repository.
+
+**Table of contents:**
+
+- [Before you begin](#before-you-begin)
+- [Step 1: Configure the `scimsession` credentials](#step-1-configure-the-scimsession-credentials)
+- [Step 2: Create the container app](#step-2-create-the-container-app)
+- [Step 3: Configure and deploy your SCIM bridge](#step-3-configure-and-deploy-your-scim-bridge)
+- [Step 4: Connect your identity provider](#step-4-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Resource recommendations](#appendix-resource-recommendations)
+- [Appendix: Customize your deployment](#appendix-customize-your-deployment)
+- [Get help](#get-help)
+
+## Before you begin
+
+Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there. You'll also need the following:
+
+- An Azure account with permissions to create a Container App.
+
+> [!NOTE]
+> If you don't have a Azure account, you can sign up for a free trial with starting credit: https://azure.microsoft.com/en-us/free/
+
+- [Azure CLI tool](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) with the ContainerApp Extension or access to the [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/quickstart) if you'd like to deploy using the command line.
```

Comment:
I guess this is not really a true requirement here, since we are guiding how to deploy with the portal GUI. 
```suggestion
```